packages/react-composites/tests/browser/call/Localization.test.ts

```diff
@@ -4,21 +4,22 @@
 import { test } from './fixture';
 import { loadPageWithPermissionsForCalls, waitForCallCompositeToLoad, loadCallScreen } from '../common/utils';
 import { expect } from '@playwright/test';
+import { v1 as generateGUID } from 'uuid';
 
 test.describe('Localization tests', async () => {
-  test.beforeEach(async ({ pages }) => {
-    for (const page of pages) {
-      // Ensure any previous call users from prior tests have left the call
-      await page.reload();
-    }
-  });
-
   test('Configuration page title and participant button in call should be localized', async ({
     serverUrl,
     users,
     testBrowser
   }) => {
-    const page = await loadPageWithPermissionsForCalls(testBrowser, serverUrl, users[0], { useFrlocale: 'true' });
+    // Run this test in an isolated thread to ensure the test is totally isolated.
```

Comment:
Please add a comment here explaining why this test is so special that it needs its own call (if you know).

Feel free to link to the PR that didn't work and state that this is a hail mary.

Reply:
Didn't help either way, still failed with the same error. Closing this PR. https://github.com/Azure/communication-ui-library/pull/855/checks?check_run_id=3747775895